rir/src/compiler/translations/rir_2_pir/rir_2_pir_compiler.cpp

```diff
@@ -203,6 +203,12 @@ void Rir2PirCompiler::optimizeModule() {
 
 #ifdef ENABLE_SLOWASSERT
                 Verify::apply(v);
+#else
+#ifdef FULLVERIFIER
```

Comment:
this might be a bit confusing, since SLOWASSERT which sounds less strict than FULLVERIFIER actually takes precedence.

I would reorder the cases.